Dockerfile.deps

```diff
@@ -80,6 +80,14 @@ RUN JPEG_TURBO_VERSION=1.5.3 && \
     make -j"$(grep ^processor /proc/cpuinfo | wc -l)" install 2>&1 >/dev/null && \
     rm -rf /libjpeg-turbo-${JPEG_TURBO_VERSION}
 
+
+# Clang
+RUN CLANG_VERSION=6.0.1 && \
```

Comment:
I couldn't make --strip 1 to work with this approach, and it is needed because there is top-level directory containing all the bin/ lib/ etc. And if I skip it, I would still need to move the contents of unpacked dir.